frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterSet, FilterValue, RangeFilterValue } from "../../models/paged_items";
+
+interface FiltersInfosProps {
+    filters: FilterSet
+}
+
+const FiltersInfos = ({ filters }: FiltersInfosProps) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
+    const getValue = (key: any) => {
+        const filterValue: FilterValue = filters[key].value ?? {}
+        const description: any = filters[key].description
```

Comment:
The type here is `FilterDescription | undefined`. You'll need to add a null check somewhere to handle the case where the description is undefined.